ydb/docs/en/core/reference/languages-and-apis/ado-net/connection_parameters.md

```diff
@@ -0,0 +1,49 @@
+# Connection parameters {#connection-parameters}
+
+To connect to a database, the application provides a connection string which specifies parameters such as the host, the user, the password, etc. Connection strings have the form `keyword1=value; keyword2=value;`. For more information, [see the official doc page on connection strings](https://learn.microsoft.com/en-us/dotnet/framework/data/adonet/connection-strings).
+
+All available connection parameters are defined as properties in the `YdbConnectionStringBuilder`.
+
+Below are the connection string parameters which Ydb.Sdk.Ado understands.
+
+## Basic connection
+
+| Parameter         | Description                                                                                                 | Default value |
+|-------------------|-------------------------------------------------------------------------------------------------------------|---------------|
+| `Host`            | Specifies the {{ ydb-short-name }} server host                                                              | `localhost`   |
+| `Port`            | Specifies the {{ ydb-short-name }}  server port                                                             | `2136`        |
+| `Database`        | Specifies the database name                                                                                 | `/local`      |
+| `User`            | Specifies the username                                                                                      | Not defined   |
+| `Password`        | Specifies the user password                                                                                 | Not defined   |
+
+## Security and encryption
+
+| Parameter         | Description                                                                                                 | Default value |
+|-------------------|-------------------------------------------------------------------------------------------------------------|---------------|
+| `UseTls`          | Indicates whether to use the TLS protocol (`grpcs` or `grpc`)                                               | `false`       |
+| `RootCertificate` | Specifies the path to the trusted server TLS certificate. If this parameter is set, `UseTls` will be `true` | Not defined   |
```

Comment:
Unclear: will it be true by default, or will it always be true and can't be overridden?